docs/src/main/asciidoc/transaction.adoc

```diff
@@ -362,16 +362,21 @@ NOTE: The `event` object represents the transaction ID, and defines `toString()`
 TIP: In listener methods, you can access more information about the transaction in progress by accessing the `TransactionManager`,
 which is a CDI bean and can be ``@Inject``ed.
 
-== Configuring transaction log to be stored in a DataSource
+[[jdbcstore]]
+== Configure the transaction log to be stored in a dataSource
```

Comment:
Why did you call it a dataSource instead of a database. I ask because a dataSource can be backed by other storage such as a filesystem, for example, and in this case we'd recommend that users use the file or journal stores directly.

Reply:
I think its because Narayana is mention in a Datasource guide section, so calling it like that creates the logical follow-up.
The usage of the database vs datasource (or even data source) is a little problem overall all our docs.